pkg/health/report.go

```diff
@@ -0,0 +1,37 @@
+package health
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/cloud-run-release-operator/pkg/config"
+)
+
+// StringReport returns a human-readable report of the diagnosis.
+//
+// The returned string has the format:
+// status: healthy
+// metrics:
+// - request-latency[p99]: 500.00 (needs 750.0)
+// - request-count: 800 (needs 1000)
+func StringReport(healthCriteria []config.Metric, diagnosis Diagnosis) string {
+	report := fmt.Sprintf("status: %s\n", diagnosis.OverallResult.String())
```

Comment:
what if we also added the time we wrote this down?
does the user have any way of telling when was this status report for?

Reply:
We could, but I'd probably have a function `setHealthReportAnnotation(report string)` in `rollout` since there are cases in which this function is not called (e.g. new candidate). In that function, we can append the time at the end of any report.

I will defer this after #57 since we would need the mocked time introduced there. Doing it here would be double work and would make this PR much longer.